Yank/repex.py

```diff
@@ -1181,7 +1237,9 @@ class while the simulation is running. This reads the SamplerStates of every
     # Constructors.
     # -------------------------------------------------------------------------
 
-    def __init__(self, mcmc_moves=None, number_of_iterations=1, replica_mixing_scheme='swap-all'):
+    def __init__(self, mcmc_moves=None, number_of_iterations=1, replica_mixing_scheme='swap-all',
+                 online_analysis_interval=None, online_analysis_target_error=1.0,
```

Comment:
This is the target error per *phase* right? If we want errors ~0.2 kcal/mol by default, that's allowing something like 0.2 kT (0.14 kcal/mol) error in each phase.